client/my-sites/site-settings/composing/posts-per-page.jsx

```diff
@@ -0,0 +1,65 @@
+/**
+ * External dependencies
+ */
+import React, { PropTypes } from 'react';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import FormFieldset from 'components/forms/form-fieldset';
+import FormLabel from 'components/forms/form-label';
+import FormInput from 'components/forms/form-text-input';
+import FormSettingExplanation from 'components/forms/form-setting-explanation';
+
+const PostsPerPage = ( {
+	fields,
+	onChangeField,
+	eventTracker,
+	uniqueEventTracker,
+	isRequestingSettings,
+	isSavingSettings,
+	translate
+} ) => {
+	if ( 'undefined' === typeof fields.posts_per_page ) {
```

Comment:
I see this is moved verbatim from the previous location. Did you consider whether this is the best check to make here?